express-get-json/server.ts

```diff
@@ -0,0 +1,48 @@
+import express from "express";
+
+
+interface Person {
```

Comment:
Personally, I dislike `interface` and always use `type`. I know the TS website implies the opposite, but all code I write and most code I interact with uses `type`.

There are a few rare cases where `interface` is useful, but almost always `type` provides more flexibility and convenience.